src/test/java/it/geosolutions/httpproxy/HttpProxyIntegrationTests.java

```diff
@@ -206,6 +207,27 @@ public void testPUTUsingWireMock() throws IOException {
         }
     }
 
+    /**
+     * Validates that HTTP URL pot set to 80 if no port is specified
+     */
+    @Test
+    public void testHttpDefaultPort() throws IOException {
+        String urlStr = "http://localhost:" + "/geostore/users/5";
+        URL url = Utils.buildURL(urlStr);
+        Assert.assertEquals(url.getPort(), Utils.DEFAULT_HTTP_PORT);
```

Comment:
I would check the final url entirely so we test that also request path is preserved. Maybe adding also some random request params.

Reply:
done